test/oli_web/live/sections/edit_live_test.exs

```diff
@@ -196,5 +196,46 @@ defmodule OliWeb.Sections.EditLiveTest do
              |> element("#section_display_curriculum_item_numbering")
              |> render() =~ "checked"
     end
+
+    test "update section with a long title shows an error alert", %{conn: conn, section: section} do
+      {:ok, view, _html} = live(conn, "/authoring/products/#{section.slug}")
+
+      long_title =
+        "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum"
+
+      view
+      |> element("form[phx-submit=\"save\"")
+      |> render_submit(%{section: %{title: long_title}})
+
+      assert view
+             |> element("div.alert.alert-danger")
+             |> render() =~
+               "Couldn&#39;t update product title"
+
+      assert view
```

Comment:
I think this assert is ok, but what do you think about testing that the actual product was not updated?

Reply:
I think that is an excellent idea!